pkg/box/box.go

```diff
@@ -208,6 +208,21 @@ type SnapshotTarget struct {
 
 	// ReadyAddress is a URL to ping before marking the devenv as ready
 	ReadyAddress string `yaml:"readyAddress"`
+
+	// RestoreSteps defines which steps are used for restore
+	RestoreSteps []RestoreStep `yaml:"restore_steps"`
+}
+
+// RestoreStep is the defn for a restore step
+type RestoreStep struct {
+	// Description allows to specify what's this intended to exactly
```

Comment:
"intended to exactly" doesn't read well, maybe something like "describes what this restore step does"?